.travis.yml

```diff
@@ -19,94 +19,40 @@
 sudo: true
 dist: trusty
 language: python
-jdk:
-  - oraclejdk8
-services:
-  - cassandra
-  - mongodb
-  - mysql
-  - postgresql
-  - rabbitmq
-addons:
-  apt:
-    packages:
-      - slapd
-      - ldap-utils
-      - openssh-server
-      - mysql-server-5.6
-      - mysql-client-core-5.6
-      - mysql-client-5.6
-      - krb5-user
-      - krb5-kdc
-      - krb5-admin-server
-      - oracle-java8-installer
-  postgresql: "9.2"
-python:
-  - "2.7"
-  - "3.5"
 env:
   global:
+    - DOCKER_COMPOSE_VERSION=1.20.0
     - SLUGIFY_USES_TEXT_UNIDECODE=yes
     - TRAVIS_CACHE=$HOME/.travis_cache/
-    - KRB5_CONFIG=/etc/krb5.conf
-    - KRB5_KTNAME=/etc/airflow.keytab
-    # Travis on google cloud engine has a global /etc/boto.cfg that
-    # does not work with python 3
-    - BOTO_CONFIG=/tmp/bogusvalue
   matrix:
+    - TOX_ENV=flake8
     - TOX_ENV=py27-backend_mysql
     - TOX_ENV=py27-backend_sqlite
     - TOX_ENV=py27-backend_postgres
-    - TOX_ENV=py35-backend_mysql
-    - TOX_ENV=py35-backend_sqlite
-    - TOX_ENV=py35-backend_postgres
-    - TOX_ENV=flake8
+    - TOX_ENV=py35-backend_mysql PYTHON_VERSION=3
```

Comment:
@gerardo I am currently updating the Airflow CI to point to 3.6 instead of python 3.5 

But the way Python Version is passed using this environment variable, can you help me figure out how this can be achieved?

Reply:
Also, we need to keep in mind that in future we would need to test both Python3.6 and Python3.7

Reply:
https://github.com/apache/incubator-airflow/pull/3815

Reply:
@kaxil give we're using tox for this, it should be a matter of just adding/updating the runtimes inside the tox file [here](https://github.com/apache/incubator-airflow/blob/master/tox.ini#L20) and [here](https://github.com/apache/incubator-airflow/blob/master/tox.ini#L40) and then pass the desired python version through the Travis script as we do now.

There's an extra check inside the [run-ci script](https://github.com/apache/incubator-airflow/blob/master/scripts/ci/run-ci.sh#L29) to use the correct pip binary (pip vs pip3), but this shouldn't be affected, as that only cares about the mayor version.